packages/stable/src/__tests__/api/files.int.spec.ts

```diff
@@ -20,11 +21,39 @@ import {
 
 const testfile = join(__dirname, '../test3dFile.fbx');
 
-describe.skip('Files integration test', () => {
```

Comment:
Note these tests have been disabled since 2021, I am just un-disabling the suite so I can write my own tests and disabling all of the existing tests individually. Only diff is that it now runs at least the integration tests for instance ids but still no test for the rest of the files API, this needs to be addressed in a separate task